LibertyDSNP/spec
# Chore/sync activity content spec/1

Problem
=======
Make changes to address inconsistencies between ActivityContent package and Spec

https://github.com/LibertyDSNP/activity-content/issues/1

Solution
========
Making changes to the Activity Content portion of the spec to better clarify required values and parameter names

Change summary:
--------------
- [x] Tidy, well formulated change
- [x] Rename URL properties for Associated types
- [x] Update respective examples to match
- [x] Amended Associated Type: Hash example

Steps to Verify:
----------------
1. Build spec
2. Verify changes are consistent across various attachment types - Audio, Video, Image
